applications/fastapi-bootcamp/values-idfdemo.yaml

```
@@ -0,0 +1,6 @@
+image:
+  pullPolicy: Always
+  tag: "tickets-DM-43967"
+config:
+  logLevel: "DEBUG"
+  loggingProfile: "development"
```

Must be production for Google-friendly JSON log messages. I only use development for local logs.

```suggestion
  loggingProfile: "production"
```